gammapy/estimators/flux_map.py

```diff
@@ -0,0 +1,480 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import logging
+import numpy as np
+from astropy import units as u
+from astropy.table import Table
+from astropy.coordinates import SkyCoord
+from astropy.io import fits
+
+from gammapy.maps import MapCoord, Map
+from gammapy.estimators import FluxPoints
+from gammapy.utils.table import table_from_row_data
+from gammapy.modeling.models import (
+    SkyModel,
+    PowerLawSpectralModel,
+    PointSpatialModel,
+)
+from gammapy.utils.scripts import make_path
+
+__all__ = ["FluxMap"]
+
+
+#TODO replace by ref_dnde in all code
+REQUIRED_MAPS = {
+    "dnde": ["dnde"],
+    "e2dnde": ["e2dnde"],
+    "flux": ["flux"],
+    "eflux": ["eflux"],
+    "likelihood": [
+        "dnde_ref",
+        "norm",
+    ],
+}
+
+OPTIONAL_MAPS = {
+    "dnde": ["dnde_err", "dnde_errp", "dnde_errn", "dnde_ul", "is_ul"],
+    "e2dnde": ["e2dnde_err", "e2dnde_errp", "e2dnde_errn", "e2dnde_ul", "is_ul"],
+    "flux": ["flux_err", "flux_errp", "flux_errn", "flux_ul", "is_ul"],
+    "eflux": ["eflux_err", "eflux_errp", "eflux_errn", "eflux_ul", "is_ul"],
+    "likelihood": ["norm_err", "norm_errn", "norm_errp","norm_ul", "norm_scan", "stat_scan"],
+}
+
+class FluxMap:
```

Comment:
In general If fell like  the plural `FluxMaps` is maybe more appropriate here, but I can see e.g. in the `IRFMap` we use singular as well, even if it contains in fact two maps....

Reply:
Changed to FluxMaps